platform/localization.mm

```diff
@@ -18,7 +18,7 @@
 std::string GetLocalizedBrandName(std::string const & brand)
 {
   auto const key = "brand." + brand;
-  return [NSLocalizedString(@(key.c_str()), @"") UTF8String];
+  return [NSLocalizedStringWithDefaultValue(@(key.c_str()), nil, NSBundle.mainBundle, @(brand.c_str()), @"") UTF8String];
```

Comment:
brands.txt is localized natively instead of using C++? Let's create a ticket to refactor it.

Reply:
```suggestion
  // TODO: Remove overhead of always creating NSString from brand, even if translation exists.
  return [NSLocalizedStringWithDefaultValue(@(key.c_str()), nil, NSBundle.mainBundle, @(brand.c_str()), @"") UTF8String];
```

Reply:
What kind of ticket? brands are localized the same way as types, cuisines, .. What do you want to refactor?

Reply:
- What do you want to refactor? Brands are localized the same way as types, cuisines, ..
- ToDo is useless here. Propose a solution. Otherwise we will put this kind of todos in every string -> NSString code.

Reply:
Do you think that the current localization approach is good enough? Shouldn't the C++ core load localizations instead of using native wrappers?